rcljava/src/main/cpp/org_ros2_rcljava_RCLJava.cpp

```diff
@@ -44,9 +45,47 @@ Java_org_ros2_rcljava_RCLJava_nativeCreateContextHandle(JNIEnv *, jclass)
   return context_handle;
 }
 
+bool parse_arguments(JNIEnv * env, jobject cli_args, rcl_arguments_t * arguments)
+{
+  // TODO(clalancette): we could probably make this faster by just doing these
+  // method lookups during some initialization time.  But we'd have to add a lot
+  // more infrastructure for that, and we don't expect to call
+  // 'nativeCreateNodeHandle' that often, so I think this is OK for now.
+  jclass java_util_ArrayList = static_cast<jclass>(env->FindClass("java/util/ArrayList"));
```

Comment:
nit: [FindClass](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html#FindClass) is already returning a `jclass`.
```suggestion
  jclass java_util_ArrayList = env->FindClass("java/util/ArrayList");
```

Reply:
Fixed in ad331e5